test/database.d

```diff
@@ -0,0 +1,28 @@
+import std.file;
+import dlangbot.database;
+
+@("database")
+unittest
+{
+    // Create a temporary database for this unit test only
+    enum databasePath = "var/db-test-database.s3db";
+    auto db = initDatabase(databasePath);
```

Comment:
This creates a local `Database` instance, which is never written to. However, you need to modify the global database object, which is what `updateScoreBoard` uses.

Reply:
... this is embarrassing ...

Thanks!

Reply:
This time it was on me (bug in ae with multiple databases), I pushed a version bump to the fix.